tests/mocks/test_retry.py

```diff
@@ -1,10 +1,8 @@
 """
-Test basic functionality of the Tables endpoint
+Test that requests are retried if the server is not available.
 """
-import time
 import unittest
 from unittest.mock import patch
-from urllib.error import HTTPError
 
 import requests
 import responses
```

Comment:
Shouldn't you test the chaging of the max requests as well?

Reply:
Would be nice

Reply:
I added some.

Reply:
Could one of you please merge this request? I cannot do it thanks to that codeclimate error.

Reply:
LGTM

Reply:
Sorry about that, I thought I already approved the codeclimate in the morning